src/main/java/com/minecolonies/entity/ai/citizen/guard/EntityAIRangeGuard.java

```diff
@@ -0,0 +1,300 @@
+package com.minecolonies.entity.ai.citizen.guard;
+
+import com.minecolonies.colony.jobs.JobGuard;
+import com.minecolonies.entity.ai.util.AIState;
+import com.minecolonies.entity.ai.util.AITarget;
+import net.minecraft.enchantment.EnchantmentHelper;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.IRangedAttackMob;
+import net.minecraft.entity.projectile.EntityTippedArrow;
+import net.minecraft.init.Enchantments;
+import net.minecraft.init.Items;
+import net.minecraft.init.SoundEvents;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.world.DifficultyInstance;
+import org.jetbrains.annotations.NotNull;
+
+import static com.minecolonies.entity.ai.util.AIState.*;
+
+/**
+ * Handles the AI of the guard entities.
+ */
+public class EntityAIRangeGuard extends AbstractEntityAIGuard implements IRangedAttackMob
+{
+    /**
+     * Basic delay for the next shot.
+     */
+    private static final int BASE_RELOAD_TIME = 60;
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)